overlay.d/05core/usr/lib/dracut/modules.d/35coreos-ignition/coreos-post-ignition-checks.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# See coreos-post-ignition-checks.service for more information about this script
+set -euo pipefail
+
+# Get mountpoint of boot device
+bootmnt=$(findmnt --source /dev/disk/by-label/boot -no TARGET; 2>/dev/null)
```

Comment:
This looks odd. Did you mean

```suggestion
bootmnt=$(findmnt --source /dev/disk/by-label/boot -no TARGET 2>/dev/null)
```

?

Though we need to handle the error case. I guess we can do

```suggestion
bootmnt=$(findmnt --source /dev/disk/by-label/boot -no TARGET 2>/dev/null || :)
```

We usually should avoid this idiom IMO because it can mask real issues, but I don't have a good alternative. I guess we could iterate over all mounts looking for the matching `SOURCE` source, but that would be better done in !bash.

Reply:
how about something like 

```
if [ ! -e /dev/disk/by-label/boot ]; then
    should we error here if the device doesn't exist?
fi

bootdev=$(realpath /dev/disk/by-label/boot)
mountinfo=$(findmnt --json)
bootmnt=$(jq -r ".filesystems[].children[] | select(.source == \"$bootdev\").target" <<< $mountinfo)
```